src/services/session/test/controllers/SessionIndicatorControllerSpec.js

```diff
@@ -1,189 +1,180 @@
-/*global define,describe,beforeEach,jasmine,Promise,it,spyOn,expect,waitsFor,
-    runs*/
-
-define([
-    // '../../src/controllers/SessionIndicatorController'
-], function (
-    // SessionIndicatorController
-) {
-    'use strict';
-
-    xdescribe('SessionIndicatorController', function () {
-        var $scope,
+// import SessionIndicatorController from '../SessionIndicatorController';
+
+// this would be better tested in an e2e test
```

Comment:
do these not work? it would be good to have some tests until we get e2e tests

Reply:
these are Vue component tests, so I think e2e is the better way to go.

Reply:
I agree. But we don't have e2e tests yet. I think we should at least create an issue for these if we can't fix them now.